core/src/test/java/org/apache/iceberg/DeleteFileIndexTestBase.java

```diff
@@ -50,7 +50,7 @@ public abstract class DeleteFileIndexTestBase<
 
   @Parameters(name = "formatVersion = {0}")
   public static List<Object> parameters() {
-    return Arrays.asList(2, 3);
+    return Arrays.asList(2, 3, 4);
```

Comment:
nit: Maybe we should change this to use the MAX_VERSION or all Versions parameter with a (not 1) or something

Reply:
good point, I've updated all of those places to use `MAX_VERSION` and to skip v1